src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents the postal code of a person's address.
+ */
+public class PostalCode {
```

Comment:
You can implement a method to check if the postal code is in a correct format, e.g. only number.